From 6e3ff96ee617c2972aba12073388cee95e49df09 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@freescale.com>
Date: Thu, 29 Oct 2015 22:03:32 +0800
Subject: [PATCH 0645/5242] MLK-11786 ARM: dts: imx6qp: correct vddcore
 regulator input

commit  8f2782770bd3abc81ed0cfefd149f74522af2769 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX6QP SabreSD board, VDDCORE is from PFuze's SW2, this
is different from i.MX6QDL SabreSD board, which is from SW1A/B.

Signed-off-by: Anson Huang <Anson.Huang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index cfe9a7d..f62b1ab 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -54,6 +54,10 @@
 	status = "disabled";
 };
 
+&cpu0 {
+	arm-supply = <&sw2_reg>;
+};
+
 &iomuxc {
 	imx6qdl-sabresd {
 		pinctrl_usdhc2: usdhc2grp {
-- 
1.7.9.5

